docs/quick-start.md

```diff
@@ -0,0 +1,220 @@
+[https://github.com/google/protobuf/releases][protobuf-releases].
+
+* Unzip this file.
+* Update the environment variable `PATH` to include the path to the `protoc`
+  binary file.
+
+### Download the example
+
+You'll need a local copy of the example code to work through this quickstart.
+Download the example code from our GitHub repository (the following command
+clones the entire repository, but you just need the examples for this quickstart
+and other tutorials):
+
+```sh
+$ # Clone the repository at the latest release to get the example code:
+$ git clone -b 1.0.0-alpha.6 https://github.com/grpc/grpc-swift
+$ # Navigate to the repository
+$ cd grpc-swift/
+```
+
+## Run a gRPC application
+
+From the `grpc-swift` directory:
+
+1. Compile and run the server
+
+   ```sh
+   $ swift run HelloWorldServer
+   server started on port 52200
+   $ # Note: the port may be different on your machine.
+   ```
+
+2. In another terminal, compile and run the client
+
+   ```sh
+   $ swift run HelloWorldClient 52200
+   ```
+
+Congratulations! You've just run a client-server application with gRPC.
+
+## Update a gRPC service
+
+Now let's look at how to update the application with an extra method on the
+server for the client to call. Our gRPC service is defined using protocol
+buffers; you can find out lots more about how to define a service in a `.proto`
+file in [What is gRPC?][grpc-guides]. For now all you need to know is that both
+the server and the client "stub" have a `SayHello` RPC method that takes a
+`HelloRequest` parameter from the client and returns a `HelloReply` from the
+server, and that this method is defined like this:
+
+```proto
+// The greeting service definition.
+service Greeter {
+  // Sends a greeting
+  rpc SayHello (HelloRequest) returns (HelloReply) {}
+}
+
+// The request message containing the user's name.
+message HelloRequest {
+  string name = 1;
+}
+
+// The response message containing the greetings
+message HelloReply {
+  string message = 1;
+}
+```
+
+Let's update this so that the `Greeter` service has two methods. Edit
+`Sources/Examples/HelloWorld/Model/helloworld.proto` and update it with a new
+`SayHelloAgain` method, with the same request and response types:
+
+```proto
+// The greeting service definition.
+service Greeter {
+  // Sends a greeting
+  rpc SayHello (HelloRequest) returns (HelloReply) {}
+  // Sends another greeting
+  rpc SayHelloAgain (HelloRequest) returns (HelloReply) {}
+}
+
+// The request message containing the user's name.
+message HelloRequest {
+  string name = 1;
+}
+
+// The response message containing the greetings
+message HelloReply {
+  string message = 1;
+}
+```
+
+(Don't forget to save the file!)
+
+### Update and run the application
+
+We need to regenerate
+`Sources/Examples/HelloWorld/Model/helloworld.grpc.swift`, which
+contains our generated gRPC client and server classes.
+
+From the `grpc-swift` directory run
+
+```sh
+make generate-helloworld
```

Comment:
I wonder if we should specify the `protoc` command here, so that users could better adapt that command to their own project?

Reply:
I think - as the most basic tutorial - we can leave this out. It will be included in the route guide tutorial which will be linked in the "What's next" section when it's created.

Reply:
SGTM